specification/cognitiveservices/data-plane/Language/preview/2022-07-01-preview/analyzetext.json

```diff
@@ -0,0 +1,2737 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Cognitive Language Service - Text Analysis",
```

Comment:
nit: should we still keep "/en-us/" in the reference link:
https://docs.microsoft.com/en-us/azure/cognitive-services/language-service/overview\

Reply:
No, links should not have a locale in them.